src/me/TomTheDeveloper/Events/Events.java

```diff
@@ -288,11 +289,11 @@ public void onEntityInteractEntity(PlayerInteractEntityEvent event) {
                 event.getRightClicked().setPassenger(event.getPlayer());
                 return;
             } else {
-                event.getPlayer().sendMessage(ChatManager.getSingleMessage("You-Can't-Ride-Golem-From-Somebody-Else", "You can't ride the golem of somebody else!"));
+                event.getPlayer().sendMessage(LanguageManager.getLanguageFile().get("You-Can't-Ride-Golem-From-Somebody-Else").toString().replaceAll("(&([a-f0-9]))", "\u00A7$2"));
             }
         } else {
             if (event.getRightClicked().getType() == EntityType.VILLAGER || event.getRightClicked().getType() == EntityType.IRON_GOLEM)
-                event.getPlayer().sendMessage(ChatManager.getSingleMessage("Don't-Hit-Me-With-Weapon", ChatColor.RED + "You can't hit me with a weapon. That's just rude!"));
+                event.getPlayer().sendMessage(LanguageManager.getLanguageFile().get("Don't-Hit-Me-With-Weapon").toString().replaceAll("(&([a-f0-9]))", "\u00A7$2"));
```

Comment:
You always going to write .replace() after every sentence you need to support alternative color code? That is bad practice.

Reply:
Also:
Old method:  1 method gets called with two parameters.
New method: 4 methods have to be called with three parameters in total.

=> Where is the improvement here?

Ultimate goals is to have something like: LanguageManager.getMessage("Don't-Hit-Me-With-Weapon"). 
Which is 1 method with 1 param.